From d58f22b783b9ced9c8eabdf11d48aa793b2ae9f5 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Mon, 29 Sep 2014 13:06:30 -0500
Subject: [PATCH 22/25] temp: panels disable hdmi pins (need to rewrite against
 Charles patch, but this will keep things working till i get back from san
 jose)

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 .../boot/dts/am335x-bone-panel-1240x600-24bit.dtsi | 55 +++++++++++++++++++++
 arch/arm/boot/dts/am335x-bone-panel-320x240.dtsi   | 54 +++++++++++++++++++++
 .../dts/am335x-bone-panel-480x272-bgrx_16bpp.dtsi  | 55 +++++++++++++++++++++
 arch/arm/boot/dts/am335x-bone-panel-480x272.dtsi   | 56 ++++++++++++++++++++++
 arch/arm/boot/dts/am335x-bone-panel-800x480.dtsi   | 56 ++++++++++++++++++++++
 5 files changed, 276 insertions(+)

diff --git a/arch/arm/boot/dts/am335x-bone-panel-1240x600-24bit.dtsi b/arch/arm/boot/dts/am335x-bone-panel-1240x600-24bit.dtsi
index 2fd366c..482a084 100644
--- a/arch/arm/boot/dts/am335x-bone-panel-1240x600-24bit.dtsi
+++ b/arch/arm/boot/dts/am335x-bone-panel-1240x600-24bit.dtsi
@@ -7,6 +7,7 @@
  */
 
 &ocp {
+	/* hdmi */
 	P8_11_pinmux {
 		status = "disabled";
 	};
@@ -31,12 +32,66 @@
 	P8_19_pinmux {
 		status = "disabled";
 	};
+	P8_27_pinmux {
+		status = "disabled";
+	};
+	P8_28_pinmux {
+		status = "disabled";
+	};
+	P8_29_pinmux {
+		status = "disabled";
+	};
+	P8_30_pinmux {
+		status = "disabled";
+	};
+	P8_31_pinmux {
+		status = "disabled";
+	};
+	P8_32_pinmux {
+		status = "disabled";
+	};
+	P8_33_pinmux {
+		status = "disabled";
+	};
+	P8_34_pinmux {
+		status = "disabled";
+	};
+	P8_35_pinmux {
+		status = "disabled";
+	};
+	P8_36_pinmux {
+		status = "disabled";
+	};
 	P8_37_pinmux {
 		status = "disabled";
 	};
 	P8_38_pinmux {
 		status = "disabled";
 	};
+	P8_39_pinmux {
+		status = "disabled";
+	};
+	P8_40_pinmux {
+		status = "disabled";
+	};
+	P8_41_pinmux {
+		status = "disabled";
+	};
+	P8_42_pinmux {
+		status = "disabled";
+	};
+	P8_43_pinmux {
+		status = "disabled";
+	};
+	P8_44_pinmux {
+		status = "disabled";
+	};
+	P8_45_pinmux {
+		status = "disabled";
+	};
+	P8_46_pinmux {
+		status = "disabled";
+	};
 };
 
 &lcdc {
diff --git a/arch/arm/boot/dts/am335x-bone-panel-320x240.dtsi b/arch/arm/boot/dts/am335x-bone-panel-320x240.dtsi
index 9643f0e..06d1596 100644
--- a/arch/arm/boot/dts/am335x-bone-panel-320x240.dtsi
+++ b/arch/arm/boot/dts/am335x-bone-panel-320x240.dtsi
@@ -8,12 +8,66 @@
 
 &ocp {
 	/* hdmi */
+	P8_27_pinmux {
+		status = "disabled";
+	};
+	P8_28_pinmux {
+		status = "disabled";
+	};
+	P8_29_pinmux {
+		status = "disabled";
+	};
+	P8_30_pinmux {
+		status = "disabled";
+	};
+	P8_31_pinmux {
+		status = "disabled";
+	};
+	P8_32_pinmux {
+		status = "disabled";
+	};
+	P8_33_pinmux {
+		status = "disabled";
+	};
+	P8_34_pinmux {
+		status = "disabled";
+	};
+	P8_35_pinmux {
+		status = "disabled";
+	};
+	P8_36_pinmux {
+		status = "disabled";
+	};
 	P8_37_pinmux {
 		status = "disabled";
 	};
 	P8_38_pinmux {
 		status = "disabled";
 	};
+	P8_39_pinmux {
+		status = "disabled";
+	};
+	P8_40_pinmux {
+		status = "disabled";
+	};
+	P8_41_pinmux {
+		status = "disabled";
+	};
+	P8_42_pinmux {
+		status = "disabled";
+	};
+	P8_43_pinmux {
+		status = "disabled";
+	};
+	P8_44_pinmux {
+		status = "disabled";
+	};
+	P8_45_pinmux {
+		status = "disabled";
+	};
+	P8_46_pinmux {
+		status = "disabled";
+	};
 };
 
 &lcdc {
diff --git a/arch/arm/boot/dts/am335x-bone-panel-480x272-bgrx_16bpp.dtsi b/arch/arm/boot/dts/am335x-bone-panel-480x272-bgrx_16bpp.dtsi
index bdd05af..50796aa 100644
--- a/arch/arm/boot/dts/am335x-bone-panel-480x272-bgrx_16bpp.dtsi
+++ b/arch/arm/boot/dts/am335x-bone-panel-480x272-bgrx_16bpp.dtsi
@@ -7,12 +7,67 @@
  */
 
 &ocp {
+	/* hdmi */
+	P8_27_pinmux {
+		status = "disabled";
+	};
+	P8_28_pinmux {
+		status = "disabled";
+	};
+	P8_29_pinmux {
+		status = "disabled";
+	};
+	P8_30_pinmux {
+		status = "disabled";
+	};
+	P8_31_pinmux {
+		status = "disabled";
+	};
+	P8_32_pinmux {
+		status = "disabled";
+	};
+	P8_33_pinmux {
+		status = "disabled";
+	};
+	P8_34_pinmux {
+		status = "disabled";
+	};
+	P8_35_pinmux {
+		status = "disabled";
+	};
+	P8_36_pinmux {
+		status = "disabled";
+	};
 	P8_37_pinmux {
 		status = "disabled";
 	};
 	P8_38_pinmux {
 		status = "disabled";
 	};
+	P8_39_pinmux {
+		status = "disabled";
+	};
+	P8_40_pinmux {
+		status = "disabled";
+	};
+	P8_41_pinmux {
+		status = "disabled";
+	};
+	P8_42_pinmux {
+		status = "disabled";
+	};
+	P8_43_pinmux {
+		status = "disabled";
+	};
+	P8_44_pinmux {
+		status = "disabled";
+	};
+	P8_45_pinmux {
+		status = "disabled";
+	};
+	P8_46_pinmux {
+		status = "disabled";
+	};
 };
 
 &lcdc {
diff --git a/arch/arm/boot/dts/am335x-bone-panel-480x272.dtsi b/arch/arm/boot/dts/am335x-bone-panel-480x272.dtsi
index aa02625..f0766f3 100644
--- a/arch/arm/boot/dts/am335x-bone-panel-480x272.dtsi
+++ b/arch/arm/boot/dts/am335x-bone-panel-480x272.dtsi
@@ -7,12 +7,68 @@
  */
 
 &ocp {
+	/* hdmi */
+	P8_27_pinmux {
+		status = "disabled";
+	};
+	P8_28_pinmux {
+		status = "disabled";
+	};
+	P8_29_pinmux {
+		status = "disabled";
+	};
+	P8_30_pinmux {
+		status = "disabled";
+	};
+	P8_31_pinmux {
+		status = "disabled";
+	};
+	P8_32_pinmux {
+		status = "disabled";
+	};
+	P8_33_pinmux {
+		status = "disabled";
+	};
+	P8_34_pinmux {
+		status = "disabled";
+	};
+	P8_35_pinmux {
+		status = "disabled";
+	};
+	P8_36_pinmux {
+		status = "disabled";
+	};
 	P8_37_pinmux {
 		status = "disabled";
 	};
 	P8_38_pinmux {
 		status = "disabled";
 	};
+	P8_39_pinmux {
+		status = "disabled";
+	};
+	P8_40_pinmux {
+		status = "disabled";
+	};
+	P8_41_pinmux {
+		status = "disabled";
+	};
+	P8_42_pinmux {
+		status = "disabled";
+	};
+	P8_43_pinmux {
+		status = "disabled";
+	};
+	P8_44_pinmux {
+		status = "disabled";
+	};
+	P8_45_pinmux {
+		status = "disabled";
+	};
+	P8_46_pinmux {
+		status = "disabled";
+	};
+	/* fixme: what are you? */
 	P9_27_pinmux {
 		status = "disabled";
 	};
diff --git a/arch/arm/boot/dts/am335x-bone-panel-800x480.dtsi b/arch/arm/boot/dts/am335x-bone-panel-800x480.dtsi
index 7b8c053..0501a00 100644
--- a/arch/arm/boot/dts/am335x-bone-panel-800x480.dtsi
+++ b/arch/arm/boot/dts/am335x-bone-panel-800x480.dtsi
@@ -7,12 +7,68 @@
  */
 
 &ocp {
+	/* hdmi */
+	P8_27_pinmux {
+		status = "disabled";
+	};
+	P8_28_pinmux {
+		status = "disabled";
+	};
+	P8_29_pinmux {
+		status = "disabled";
+	};
+	P8_30_pinmux {
+		status = "disabled";
+	};
+	P8_31_pinmux {
+		status = "disabled";
+	};
+	P8_32_pinmux {
+		status = "disabled";
+	};
+	P8_33_pinmux {
+		status = "disabled";
+	};
+	P8_34_pinmux {
+		status = "disabled";
+	};
+	P8_35_pinmux {
+		status = "disabled";
+	};
+	P8_36_pinmux {
+		status = "disabled";
+	};
 	P8_37_pinmux {
 		status = "disabled";
 	};
 	P8_38_pinmux {
 		status = "disabled";
 	};
+	P8_39_pinmux {
+		status = "disabled";
+	};
+	P8_40_pinmux {
+		status = "disabled";
+	};
+	P8_41_pinmux {
+		status = "disabled";
+	};
+	P8_42_pinmux {
+		status = "disabled";
+	};
+	P8_43_pinmux {
+		status = "disabled";
+	};
+	P8_44_pinmux {
+		status = "disabled";
+	};
+	P8_45_pinmux {
+		status = "disabled";
+	};
+	P8_46_pinmux {
+		status = "disabled";
+	};
+	/* fixme: what are you? */
 	P9_22_pinmux {
 		status = "disabled";
 	};
-- 
2.1.3

